api/src/main/kotlin/com/openlattice/organizations/external/SqlDataType.kt

```diff
@@ -0,0 +1,68 @@
+package com.openlattice.organizations.external
+
+import com.openlattice.postgres.PostgresDatatype
+import java.util.*
+
+/**
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+enum class SqlDataType {
```

Comment:
Going to delete this one. There's too much variance across data types so will just pass through the strings since we don't need the enums.